README.md

```diff
@@ -166,13 +174,13 @@ Also, note that this will [not work](https://github.com/sharkdp/bat/issues/1145)
 
 The [`prettybat`](https://github.com/eth-p/bat-extras/blob/master/doc/prettybat.md) script is a wrapper that will format code and print it with `bat`.
 
-
 ## Installation
 
 [![Packaging status](https://repology.org/badge/vertical-allrepos/bat.svg)](https://repology.org/project/bat/versions)
 
 ### On Ubuntu (using `apt`)
-*... and other Debian-based Linux distributions.*
+
+_... and other Debian-based Linux distributions._
```

Comment:
Is there any particular reason that you're replacing the `*` emphasis character here with `_`? If I recall correctly, they mean the same thing in every Markdown specification except Slack/Discord.